build.yaml

```diff
@@ -1204,3 +1203,55 @@ job_type:
                    *package_jenkins_osx_yosemite_box ]
           }
       timeout: 120
+
+#-----------------------------------------------------------------------------#
+# View definitions below this point
+#-----------------------------------------------------------------------------#
+
+views:
+  trial:
+    description: 'All trial jobs'
+    regex: 'run_(?i)trial_.*.*'
+  acceptance:
+    description: 'All Acceptance jobs'
+    regex: 'run_(?i)acceptance_.*.*'
+  client:
+    description: 'All client instalation jobs'
+    regex: 'run_(?i)client_.*.*'
+  code_quality:
+    description: 'All code quality jobs'
+    regex: 'run_(?i)lint.*.*'
+  docs:
+    description: 'Documentation jobs'
+    regex: 'run_(?i)sphinx.*.*'
+  cinder:
+    description: 'cinder jobs'
+    regex: '.*_(?i)cinder_.*.*'
+  ebs:
+    description: 'ebs jobs'
+    regex: '.*_(?i)ebs_.*.*'
```

Comment:
let me try it out.

When browsing do you find the 'AWS' and 'Rackspace' usefull ?
or could I simply drop them ?